### PR TITLE
rename package path in Dockerfile

### PR DESCRIPTION
**Pull request by sigua-cs**

### CODE REVIEW

**Comment by gontarzpawel** [comment]
There's a need for rebase @sigua-cs

File: Dockerfile [hunk 1]
```diff
@@ -2,14 +2,14 @@ FROM golang:1.13-alpine AS build
 
 RUN apk add --update zstd-static zstd-dev make gcc musl-dev git
 RUN go get golang.org/x/lint/golint
-RUN mkdir -p /go/src/github.com/Vertamedia/chproxy
-WORKDIR /go/src/github.com/Vertamedia/chproxy
+RUN mkdir -p /go/src/github.com/contentsqure/chproxy
```
Comment (pixelastic) [inline Dockerfile]:
```suggestion
RUN mkdir -p /go/src/github.com/contentsquare/chproxy
```

**Comment by gontarzpawel** [comment]
@sigua-cs could you rebase and complete this PR?

**Comment by sigua-cs** [comment]
@Garnek20 I'm closing this PR because there are no changes left after the rebase

**Pull request closed without merging**